PROTOCOL.md

```diff
@@ -620,6 +620,8 @@ A delta file can optionally contain additional provenance information about what
 
 Implementations are free to store any valid JSON-formatted data via the `commitInfo` action.
 
+When In-Commit Timestamps are enabled, writers are required to include a commitInfo action with every commit, which must include the `inCommitTimestamp` field.
```

Comment:
Cross link `In-Commit Timestamps` to right section.

Reply:
Do we also enforce commitInfo to be the first line in commit file as part of this feature?